### PR TITLE
add css for multi root elements

### PR DESCRIPTION
**Pull request by Pistacchione**
Hi,

I made a small change to the css to display multi root elements correctly.
Thanks for this great components!

Matteo

### CODE REVIEW

File: src/components/tree.vue [hunk 1]
```diff
@@ -207,7 +207,10 @@ export default {
         throw new ReferenceError('newNode is required but undefined')
       }
       if (typeof newNode === 'string') {
-        addnode = {title: newNode}
+        addnode = {
+          id: (Math.random().toString(36)+'00000000000000000').slice(2, 7), // Generate random id of 5 characters
```
Comment (halower) [inline src/components/tree.vue]: if u use a random id, then no initialization ID how to deal with?

Comment (halower) [reply src/components/tree.vue]: if the ID may exist may not exist, then the existence of id here is what is the meaning?

Comment (halower) [reply src/components/tree.vue]: I tend to the user's own incoming ID， and you

**Comment by Pistacchione** [comment]
Sorry, this pull request include also the generation of ID. I will made a brand new pull request with only che CSS part

**Pull request closed without merging**